### PR TITLE
75 deploy performance test

### PR DESCRIPTION
**Pull request by sinev-valentine**
deploying erc20 contracts for performance tests

### CODE REVIEW

File: evm_loader/performance/run.py [hunk 1]
```diff
@@ -0,0 +1,589 @@
...
+    assert(data[:1] == b'\x07')  # 7 means OnEvent
+    assert(data[1:21] == factory_eth)
+    assert(data[21:29] == bytes().fromhex('%016x' % 1)[::-1])  # topics len
+    hash = data[61:93]
+    return hash
+
+
+def get_trx(contract_eth, caller, caller_eth, input, pr_key):
+    tx = {'to': contract_eth, 'value': 1, 'gas': 1, 'gasPrice': 1,
+        'nonce': getTransactionCount(client, caller), 'data': input, 'chainId': chain_id}
```
Comment (s-medvedev) [inline evm_loader/performance/run.py:148]: You must be able to generate multiple transactions for one account. Therefore, it is worth keeping the expected nonce and using it. If this information is not available, then ask the node.

File: evm_loader/performance/run.py [hunk 2]
```diff
@@ -0,0 +1,589 @@
...
+    return mint_confirm(receipt_list, sum)
+
+
+def create_accounts(args):
+    instance = init_wallet()
+    instance.init()
+
+    receipt_list = {}
+    for i in range(args.count):
+        pr_key = w3.eth.account.from_key(random.randbytes(32))
```
[the remark this answers is not in the document]
Comment (sinev-valentine) [reply evm_loader/performance/run.py:355]: fixed

File: evm_loader/performance/sender/src/main.rs [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+                .default_value("AeTXaphqg264q2Bf1iqnLMWNiyykrfK13cKDR6WBHLGY")
+                // .default_value("Bn5MgusJdV4dhZYrTMXCDUNUfD69SyJLSXWwRk8sdp3x")
+                .help("Pubkey for evm_loader contract")
+        ).arg(
+        Arg::with_name("transactions_file")
+            .value_name("TRANSACTIONS_FILEPATH")
+            .takes_value(true)
+            .required(true)
+            .help("/path/to/transactions.json")
+            .default_value("/home/user/CLionProjects/cyber-core/neon-evm/evm_loader/performance/transactions.json1"),
```
[the remark this answers is not in the document]
Comment (sinev-valentine) [reply evm_loader/performance/sender/src/main.rs:199]: fixed

File: evm_loader/performance/run.py [hunk 4]
```diff
@@ -0,0 +1,589 @@
...
+    (recent_blockhash, blockhash_time) = get_block_hash()
+    start = time.time()
+    total = 0
+    trx_times = []
+    cycle_times = []
+    for line in eth_trx:
+        rec = json.loads(line)
+
+        cycle_start = time.time()
+        total = total + 1
```
[the remark this answers is not in the document]
Comment (sinev-valentine) [reply evm_loader/performance/run.py:479]: fixed

File: evm_loader/performance/run.py [hunk 5]
```diff
@@ -0,0 +1,589 @@
...
+
+def verify_trx(args):
+    verify = open(verify_file+args.postfix, 'r')
+    total = 0
+    event_error = 0
+    receipt_error = 0
+    for line in verify:
+        total = total + 1
+        if total > args.count:
+            break
```
[the remark this answers is not in the document]
Comment (sinev-valentine) [reply evm_loader/performance/run.py:524]: fixed

File: evm_loader/performance/run.py [hunk 6]
```diff
@@ -0,0 +1,589 @@
...
+        success = False
+        (erc20_eth, payer_eth, receiver_eth, receipt) = json.loads(line)
+        # confirm_transaction(client, receipt, sleep_time=0.1)
+        res = client.get_confirmed_transaction(receipt)
+        if res['result'] == None:
+            receipt_error = receipt_error + 1
+            print(success)
+        else:
+            try:
+                success = check_transfer_event(res['result'], erc20_eth, payer_eth, receiver_eth, transfer_sum, b'\x12')
```
[the remark this answers is not in the document]
Comment (sinev-valentine) [reply evm_loader/performance/run.py:534]: the function check_transfer_event() was updated.
I couldn't reproduce event error

File: evm_loader/performance/run.py (hunk 1 above)
Comment (sinev-valentine) [reply evm_loader/performance/run.py]: fixed

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**